### DISCUSSION TITLE
HTML in speaker notes

### DISCUSSION THREAD

**Discussion by genaumann** [marp-team/marp#502] [Q&A]
Hi,

thank you for your work, I really like the Marp ecosystem.
I use the speaker notes a lot, it would be great if the speaker notes support html tags.

It would be great if you are able to define notes like this:

```md
<!-- This is a speaker note with a [Link]("https://google.de") -->
``` 

or like this:

```md
<!-- This is a speaker note with a <a href="https://google.de">Link</a> -->
``` 

I have tried to implement this with a plugin, but this seems very complicated.

**Comment by viadee-BiM** [comment]
Just to add my vote: My team and I would also be super interested in this!
